CodingGuidelines.md

```diff
@@ -141,28 +350,42 @@ private void Update()
 
 ---
 
-Public classes, structs, and enums should all go in their own files.
+Cache references to materials, do not call the ".material" each time.
```

Comment:
Add a mention to using .sharedMaterial as an option for setting materials on objects?